### PR TITLE
Add Region field to cards

### PR DESCRIPTION
**Pull request by dhaselhan**
* Move label to BaseCode and BaseCodeDto to clean things up

### CODE REVIEW

File: alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
 import { BaseCodeEntity } from '../../../common/entities/base.code.entity';
 
 @Entity()
-export class ApplicationDecisionMaker extends BaseCodeEntity {
-  @AutoMap()
-  @Column()
-  label: string;
-
-  //TODO: Why is this not getting mapped from the inherited class?
-  @AutoMap()
```
Comment (mhuseinov) [inline alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts:12]: Did you figure out what was the issue?

File: alcs-api/src/providers/typeorm/migrations/1659979182645-seed_application_region.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        ('41ab6a5e-050b-4ff6-a95d-d3161d960347', NULL, '2022-08-08 17:22:58.362584+00', '2022-08-08 17:22:58.362584+00', 'alcs-api', NULL, 'South Coast', 'SOUR', 'South Coast Region'),
+        ('46e0b281-8acf-46c2-a4c5-b6a9932efefd', NULL, '2022-08-08 17:21:32.20377+00', '2022-08-08 17:21:32.20377+00', 'alcs-api', NULL, 'Kootenay', 'KOOR', 'Kootenay Region'),
+        ('4723d359-7cad-4b25-9961-960cbf09ff7b', NULL, '2022-08-08 17:21:32.20377+00', '2022-08-08 17:21:32.20377+00', 'alcs-api', NULL, 'Interior', 'INTR', 'Interior Region'),
+        ('9504c139-3403-48fa-b92b-5eb90c05f2d1', NULL, '2022-08-08 17:21:32.20377+00', '2022-08-08 17:21:32.20377+00', 'alcs-api', NULL, 'Island', 'ISLR', 'Island Region'),
+        ('96b5e6b0-7e07-45db-a01d-8e5da60a39b9', NULL, '2022-08-08 17:21:32.20377+00', '2022-08-08 17:21:32.20377+00', 'alcs-api', NULL, 'North', 'NORR', 'North Region');
+    `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`delete from public.application_decision_maker`);
```
Comment (mhuseinov) [inline alcs-api/src/providers/typeorm/migrations/1659979182645-seed_application_region.ts:17]: application_region ?

File: alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts:12]: No, even odder is that it appears to only be an issue with ApplicationStatus. All the other code entities are returning with description, status does not whether this is added or removed.

File: alcs-api/src/providers/typeorm/migrations/1659979182645-seed_application_region.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-api/src/providers/typeorm/migrations/1659979182645-seed_application_region.ts]: Fixed

File: alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts:12]: Not sure why I though that, it appears to not work for any of them with or without this.

Comment (dhaselhan) [reply alcs-api/src/application/application-code/application-decision-maker/application-decision-maker.entity.ts:12]: Found the actual issue, description was missing a type on the entity. Fixed.

**Review by mhuseinov — APPROVED** [review]
💯

**Pull request merged**